indico/modules/events/registration/controllers/management/fields.py

```diff
@@ -133,6 +136,14 @@ def _process(self):
         if (not enabled and self.field.type == RegistrationFormItemType.field_pd and
                 self.field.personal_data_type.is_required):
             raise BadRequest
+        if not enabled:
+            # Check this field is not conditional for other fields.
+            is_conditional = (db.session.query(RegistrationFormItem)
```

Comment:
You can probably make this a `column_property` on `RegistrationFormItem` so you can just check `self.field.is_conditional` here.

Reply:
Yep, much nicer. Done in [8eb06c9](https://github.com/indico/indico/pull/6678/commits/8eb06c950c13b1a6a1ce6069f7bc503cd9214a54).

Reply:
ah right, a normal property is enough for it. Initially I had a column property in mind (where SQLAlchemy takes care of executing the query when accessed), but since we don't need to use it in a `.filter(...)` statement (for now) nor load it efficiently for multiple items, a normal property is enough